cine-flex/src/components/Header/Header.jsx

```diff
@@ -0,0 +1,62 @@
+import { useNavigate } from 'react-router-dom';
+import logo from '../../assets/logo.png';
+import styles from '../Header/Header.module.scss';
+import MenuItem from '../MenuItem/MenuItem';
+import { AppConstants } from '../../constants/app-constants';
+import { useAuth } from '../../hooks/useAuth';
+import PropTypes from 'prop-types';
+
+/**
+ * @description Returns header component 
+ * @param {*} isLoginPage - boolean
+ * @returns 
+ */
+function Header({ isLoginPage }) {
+    const navigate = useNavigate();
+    const { isAuth, authLogout } = useAuth();
+    const menuList = isAuth 
```

Comment:
could this be refactored in optimised way & for better readability

Reply:
Converted it into functions for readability